python/cli/prelude_cli/views/auth.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+from prelude_cli.views.shared import Spinner, pretty_print
+
+
+@click.group()
+@click.pass_context
+def auth(ctx):
+    """Authentication"""
+    pass
+
+
+@auth.command("login")
+@click.option("-p", "--password", required=True, type=str)
```

Comment:
https://click.palletsprojects.com/en/stable/options/#password-prompts is a nice touch (not the confirmation prompt which seems annoying)